ui/lib/replication/addon/components/replication-primary-card.js

```diff
@@ -0,0 +1,8 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  tagName: '',
+  title: null,
```

Comment:
I didn't see anything about default value best practices online from a quick search, but I'm wondering about whether we have any best practices around when to use a `null` default value as opposed a default value with the correct type (so here it would be an empty string). Any thoughts around that?

Reply:
hmm, that's a great question! so far i've followed the pattern i've seen in our app with using `null` as an empty value. i think we do that partially to prevent any unintended side effects that could come from calling stuff on an empty string. i don't feel strongly either way though -- would you find it helpful if we moved to setting default values with the correct type?